crates/oci-tar-builder/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition.workspace = true
 tar = { workspace = true }
 sha256 = { workspace = true }
 log = { workspace = true }
-oci-spec =  { workspace = true }
+oci-spec = { git="https://github.com/containers/oci-spec-rs", rev="c109fe19b2548a86f20e48b18831bf17eb72c494", features = ["runtime"] }
```

Comment:
Should this be a workspace version?

Reply:
same reasoning here https://github.com/containerd/runwasi/pull/147/files#r1346187095